chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -59,7 +59,9 @@ export interface ILiveChatWidgetProps {
     liveChatContextFromCache?: ILiveChatWidgetContext;
     contextDataStore?: IContextDataStore;
     getAuthToken?: (authClientFunction?: string) => Promise<string | null>;
+    scrollBarProps?: IScrollBarProps;
```

Comment:
Where is this used?

Reply:
This was a change long back, maybe appearing with my changes because of merging